src/current/v23.1/create-and-configure-changefeeds.md

```diff
@@ -79,16 +81,48 @@ An {{ site.data.products.enterprise }} changefeed streams row-level changes in a
 
 ### Create
 
-To create an {{ site.data.products.enterprise }} changefeed:
+To create an {{ site.data.products.enterprise }} changefeed, you can either:
+
+- [Run `CREATE CHANGEFEED`](#run-create-changefeed) for one or multiple tables to receive all changes.
+- [Use change data capture queries](#use-change-data-capture-queries) to run `CREATE CHANGEFEED` on one table to filter the change data that the changefeed emits.
```

Comment:
Think the wording is a little off here... maybe "to run `CREATE CHANGEFEED` on a single table and filter/transform the change data that the changefeed emits."